force-app/main/default/classes/credentialExamAttemptController.cls

```diff
@@ -0,0 +1,36 @@
+/**
+ * @author Dipshika Nalawade
+ * @date   04/08/2022
+ * @description controller Class containing methods  to call service class methods Credential Exam Attempt
+ */
+public with sharing class credentialExamAttemptController {
+    /*******************************************************************************************************
+      * @description Returns Credentials Exam attempts that are in progress
+      * @param userId SF id of user
+      * @return List of Credentials Exam attempts
+      */ 
+    @AuraEnabled
+    public static List<Credential_Exam_Attempt__c> getExamAttempts(String userId)
```

Comment:
Add try catch in all AuraEnabled Methods and ensure AuraHandledException is thrown in catch